json/src/lib.rs

```diff
@@ -1140,6 +1140,39 @@ pub enum PubKeyOrAddress<'a> {
     PubKey(&'a PublicKey),
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+#[serde(untagged)]
+pub enum ScanTxoutRequest {
+    Single(String),
+    Extended {
+        dest: String,
+        range: u64,
+    }
+}
+
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, Debug)]
+pub struct ScanUtxoResult {
+    pub success: bool,
```

Comment:
The `success`, `txouts`, `height` and `bestblock` fields didn't seem to exist [on 0.18](https://bitcoincore.org/en/doc/0.18.0/rpc/blockchain/scantxoutset/), which is supported by bitcoincore-rpc iiuc.

Reply:
fixed